pymc/distributions/mixture.py

```diff
@@ -438,25 +444,86 @@ def marginal_mixture_logcdf(op, value, rng, weights, *components, **kwargs):
     return mix_logcdf
 
 
-@_get_moment.register(MarginalMixtureRV)
-def get_moment_marginal_mixture(op, rv, rng, weights, *components):
+@_moment.register(MarginalMixtureRV)
+def marginal_mixture_moment(op, rv, rng, weights, *components):
     ndim_supp = components[0].owner.op.ndim_supp
     weights = at.shape_padright(weights, ndim_supp)
     mix_axis = -ndim_supp - 1
 
     if len(components) == 1:
-        moment_components = get_moment(components[0])
+        moment_components = moment(components[0])
 
     else:
         moment_components = at.stack(
-            [get_moment(component) for component in components],
+            [moment(component) for component in components],
             axis=mix_axis,
         )
 
-    moment = at.sum(weights * moment_components, axis=mix_axis)
+    mix_moment = at.sum(weights * moment_components, axis=mix_axis)
     if components[0].dtype in discrete_types:
-        moment = at.round(moment)
-    return moment
+        mix_moment = at.round(mix_moment)
+    return mix_moment
+
+
+# List of transforms that can be used by Mixture, either because they do not require
+# special handling or because we have custom logic to enable them. If new default
+# transforms are implemented, this list and function should be updated
+allowed_default_mixture_transforms = (
+    transforms.CholeskyCovPacked,
+    transforms.CircularTransform,
+    transforms.IntervalTransform,
+    transforms.LogTransform,
+    transforms.LogExpM1,
+    transforms.LogOddsTransform,
+    transforms.Ordered,
+    transforms.Simplex,
+    transforms.SumTo1,
+)
+
+
+@_default_transform.register(MarginalMixtureRV)
+def marginal_mixture_default_transform(op, rv):
+    rng, weights, *components = rv.owner.inputs
+
+    default_transforms = [
+        _default_transform(component.owner.op, component) for component in components
```

Comment:
Right now this is okay because we don't allow nested mixtures or components to be censored rvs, is that correct? Would it be a good idea to add a check or warning or comment about this? (for ourselves and users?)

Reply:
Default transforms should work with those types of distributions as well. No meed to treat them differently.

Reply:
What was your concern though?